### ISSUE TITLE
The select element in the footer of every page is not labeled properly for screen reader users

### ISSUE THREAD

**Issue by Paul-Clue** [LLK/scratch-www#7173]
### Where To Find Bug
In the footer of all pages

### Expected Behaviour
The screen reader should announce the accessible name of the element when the element is in tab-focus. 

### Actual Behaviour
The screen reader announces the following:
“English, required menu pop-up collapse, button”.

### Reproduce

- Open https://scratch.mit.edu/ in any browser on your computer
- Turn-on your screen reader of choice
- Press tab until you get to the select form-control element for language located in the footer
- Notice the screen reader gives no indication of what the select element is for

### System Details
macOs Ventura, Chrome Browser, VoiceOver (AT)
